Set/Models/SetGameModel.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+struct SetGameModel {
+    struct Card {
+        var content: CardContentModel
+        var isActive = false // true when user tap on a card
+        // I'm not sure about naming
```

Comment:
it sounds ok, every boolean should start with `is` or `should`... or something similar